app/assess/models/fund_summary.py

```diff
@@ -55,7 +55,7 @@ def create_fund_summaries(fund: Fund) -> list[FundSummary]:
 def is_after_today(date_str: str):
     """Check if the provided datetime string has passed the current datetime"""
     uk_tz = pytz.timezone("Europe/London")
-    date_format = "%Y-%m-%d %H:%M:%S"
+    date_format = "%Y-%m-%dT%H:%M:%S"
```

Comment:
This comes out as 2023-05-16T09:31:17 - is there a reason for the T? I don't see the output improvement

Reply:
Its ISO 8601 format: https://en.wikipedia.org/wiki/ISO_8601

Reply:
Ah, I see, it's being used as a calculation comparison. Works for me